foxglove/console/lib.go

```diff
@@ -145,14 +145,18 @@ func Login(ctx context.Context, client *FoxgloveClient, authDelegate AuthDelegat
 	if err != nil {
 		return "", fmt.Errorf("failed to fetch device code: %w", err)
 	}
-	fmt.Println("Enter this code in your browser: ", info.UserCode)
 	browser, err := authDelegate.openBrowser(info.VerificationUriComplete)
-	if err != nil {
-		return "", fmt.Errorf("failed to open browser: %w", err)
+	if err == nil {
+		fmt.Println("Enter this code in your browser: ", info.UserCode)
+		defer func() {
+			_ = browser.Process.Kill()
+		}()
+	} else {
+		fmt.Printf("Failed to open browser window (%s)\n", err)
```

Comment:
In my opinion, we should remove this now; we're handling the error for the user and the message doesn't help them.